### ISSUE TITLE
VROOM + ORS localhost:8080 error

### ISSUE THREAD

**Issue by endmr11** [VROOM-Project/vroom-docker#89]
I'm sure ORS is running on my 8080 port. Example url: http://localhost:8080/ors/v2/directions/driving-car?start=8.681495,49.41461&end=8.687872,49.420318

But whenever I send a request to VROOM on port 3000, which I installed with the docker image, I get the following error:
{
 "code": 3,
 "error": "Failed to connect to 0.0.0.0:8080"
}

I am making the same sample request given on the ORS original website. Can you please help? Thank you very much.

My method of installing VROOM:
 ~  docker run -dt --name vroom -p 3000:3000 -v $PWD/conf:/conf -e VROOM_ROUTER=ors ghcr.io/vroom-project/vroom-docker:v1.14.0
 
 My method of installing ORS:
 wget https://github.com/GIScience/openrouteservice/releases/download/v8.0.0/docker-compose.yml
 docker compose up -d
 
 Example Request:
 `{
    "jobs": [
        {
            "id": 1,
            "service": 300,
            "delivery": [
                1
            ],
            "location": [
                1.98465,
                48.70329
            ],
            "skills": [
                1
            ],
            "time_windows": [
                [
                    32400,
                    36000
                ]
            ]
        },
        {
            "id": 2,
            "service": 300,
            "delivery": [
                1
            ],
            "location": [
                2.03655,
                48.61128
            ],
            "skills": [
                1
            ]
        },
        {
            "id": 3,
            "service": 300,
            "delivery": [
                1
            ],
            "location": [
                2.39719,
                49.07611
            ],
            "skills": [
                2
            ]
        },
        {
            "id": 4,
            "service": 300,
            "delivery": [
                1
            ],
            "location": [
                2.41808,
                49.22619
            ],
            "skills": [
                2
            ]
        },
        {
            "id": 5,
            "service": 300,
            "delivery": [
                1
            ],
            "location": [
                2.28325,
                48.5958
            ],
            "skills": [
                14
            ]
        },
        {
            "id": 6,
            "service": 300,
            "delivery": [
                1
            ],
            "location": [
                2.89357,
                48.90736
            ],
            "skills": [
                14
            ]
        }
    ],
    "vehicles": [
        {
            "id": 1,
            "profile": "driving-car",
            "start": [
                2.35044,
                48.71764
            ],
            "end": [
                2.35044,
                48.71764
            ],
            "capacity": [
                4
            ],
            "skills": [
                1,
                14
            ],
            "time_window": [
                28800,
                43200
            ]
        },
        {
            "id": 2,
            "profile": "driving-car",
            "start": [
                2.35044,
                48.71764
            ],
            "end": [
                2.35044,
                48.71764
            ],
            "capacity": [
                4
            ],
            "skills": [
                2,
                14
            ],
            "time_window": [
                28800,
                43200
            ]
        }
    ]
}`

**Comment by jcoupey** [comment]
Hum, if you have ORS accessible from the host machine with `http://localhost:8080`,  I would not expect it to be accessible out of the box from within the VROOM container, or am I missing something?

**Comment by endmr11** [comment]
<img width="1206" alt="image" src="https://github.com/user-attachments/assets/dc046cbf-4969-4c31-8dbb-6055eddecdd6">

What path should I follow?

**Comment by jcoupey** [comment]
I don't use Docker so I wouldn't really know. I guess you have to do something so that port `8080` from the host is accessible from the container running VROOM.

**Comment by nilsnolde** [comment]
yeah, as @jcoupey suggests, you exposed the ORS port to the _host_, not other docker containers. there's a [section](https://github.com/VROOM-Project/vroom-docker?tab=readme-ov-file#routing-server-in-local-docker-container) in the readme clearly talking about that, but generally we assume basic docker(-compose) knowledge.
